lmp/jobserv.yml

```diff
@@ -299,129 +299,6 @@ triggers:
         persistent-volumes:
           bitbake: /var/cache/bitbake
 
-  - name: build-release-stable
```

Comment:
This was actually used even as part of our previous kirkstone merge (https://ci.foundries.io/projects/lmp/builds/2753/).

Reply:
> build-release-stable is currently basically a post merge run for kirkstone.

this build-release-stable is already there before we create the kirkstone branch and the last change of the job is from 3 year ago. before it is used for honister.

> This was actually used even as part of our previous kirkstone merge (https://ci.foundries.io/projects/lmp/builds/2753/).

Right but please note this is a side effect of the kirkstone branch creation. When we created the kirkstone branch we should have changed the configuration of the build-release job that follows refs/heads/main to start following the refs/heads/kirkstone.